testing/integration/otel_test.go

```diff
@@ -133,6 +133,11 @@ func TestOtelFileProcessing(t *testing.T) {
 		`"stringValue":"system.log"`, // system.log is being processed
 	})
 
+	// check `elastic-agent otel validate` command works for otel config
+	out, err := fixture.Exec(ctx, []string{"otel", "validate", "--config", cfgFilePath})
```

Comment:
I would like to see a test (here or elsewhere) that the validate command actually detects errors in otel configuration files.

That is the real value of the validate command, lets make sure it works. This test would pass even if we never called the otel validate function and just parsed the command line arguments.

Reply:
this is now addressed at https://github.com/elastic/elastic-agent/pull/4023/commits/e432f98e14709f96e2ca849df0a00f285ce3f4db

Reply:
i pulled it outside into separate function but kept within same test to spare some resources